forest/src/daemon.rs

```diff
@@ -42,6 +43,17 @@ use forest_fil_cns::composition as cns;
 #[cfg(feature = "forest_deleg_cns")]
 use forest_deleg_cns::composition as cns;
 
+// If we did not detach no existing mapping using the current configuration exists and this function is a no-op
+fn unblock_parent_process() {
+    if let Ok(shmem) = super::ipc_shmem_conf().open() {
+        let (event, _) =
+            unsafe { Event::from_existing(shmem.as_ptr()).expect("from_existing must succeed") };
+
+        event.set(EventState::Signaled).expect("set must succeed");
+        info!("Parent unblocked");
```

Comment:
is it maybe a `debug`? If it is an `info` we may need a less cryptic message for those not exposed to systems programming.

Reply:
Log removed.